srcCpp/Infrastructure_pro.cxx

```diff
@@ -10,20 +10,22 @@
 PerftestClock::PerftestClock()
 {
     clock = RTIHighResolutionClock_new();
+    if (clock == NULL) {
+        std::bad_alloc exception;
+        throw exception;
```

Comment:
Just `throw std::bad_alloc()` is customary

Reply:
Done.